examples/data-science/pandas/CDC_NDI_Mortality_Syft_Data_Owner.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<p>Comment and code does not match</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Yes, because the top commented line returns no results with the first 10k, so the second query is just there to make sure we can get some rows. I have added a comment to explain but this will go away when we can handle the full dataset.</p>